sass/navigation/_menus.scss

```diff
@@ -255,6 +255,22 @@
 			width: auto !important;
 		}
 	}
+
+	// Ubermenu
+	.ubermenu-responsive-toggle {
+		display: inline-block;
+	}
+
+	@media screen and (min-width: 960px) {
```

Comment:
@adiraomj is there any way of doing this differently. I don't think this is compatible with the UberMenu breakpoint setting.